src/tool/csv2yaml.cpp

```diff
@@ -111,6 +113,38 @@ bool process( const std::string& type, uint32 version, const std::vector<std::st
 	return true;
 }
 
+static bool instance_readdb_sub(char* str[], int columns, int current) {
+	YAML::Node node;
+
+	node["Id"] = str[0];
+	node["Name"] = str[1];
+	node["TimeLimit"] = str[2];
+	node["IdleTimeOut"] = str[3];
+
+	node["EnterMap"] = str[4];
+	node["EnterX"] = str[5];
+	node["EnterY"] = str[6];
+
+	if (columns > 7) {
+		for (int i = 0; i < columns; i++) {
```

Comment:
You need a seperate counter here in case one of the continue cases hits.
Maybe use this chance and start i with 7 and start the additional counter at 0.